joomla/joomla-cms
# Isis pagination

Pull Request for Issue #6557
#### Summary of Changes

Remove ... value from pagination as it makes no sense as reported by @spignataro
#### Testing Instructions

Set list limit to 5 on any view with many items and check the display of pagination at the bottom of the page. It should always have a value after the patch. Before the patch the middle value will always be ...
